site/_wiki/FAQ/ModelSpecific.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Model-specific FAQ
+---
+
+### Wacom CTL-x100
+
+It is possible for Wacom CTL-x100 tablets to boot in Android mode (the mode they use when interfacing with android devices like phones) instead of PC mode in some rare cases.
+
+To fix this, press the outer 2 express keys for 3-4 seconds until the LEDs change brightness.
+
+This toggles the tablet's operating mode between PC (high LED brightness) and Android mode (low LED brightness).
+
+OpenTabletDriver can only talk to the tablet in PC mode.
+
+---
+
+### I am still having problems
+
+If your tablet model is supported, please see the [General FAQ]({% link _wiki/FAQ/General.md %})
```

Comment:
Probably something like:

```suggestion
See [General FAQ]({% link _wiki/FAQ/General.md %}).
```

The original line sounds like there's nothing they can do anymore if the tablet is not supported. General FAQ has instructions on what to do in case the tablet they have isn't actually supported by OTD yet.